superset-frontend/src/features/alerts/components/AlertReportCronScheduler.tsx

```diff
@@ -77,29 +117,8 @@ export const AlertReportCronScheduler: React.FC<AlertReportCronSchedulerProps> =
               displayError={scheduleFormat === 'picker'}
               onError={onError}
             />
-          </div>
-          <div className="inline-container add-margin">
-            <Radio data-test="input" value="input" />
-            <span className="input-label">{t('CRON Schedule')}</span>
-            <StyledInputContainer
-              data-test="input-content"
-              className="styled-input"
-            >
-              <div className="input-container">
-                <Input
-                  type="text"
-                  name="crontab"
-                  ref={inputRef}
-                  style={error ? { borderColor: theme.colors.error.base } : {}}
-                  placeholder={t('CRON expression')}
-                  disabled={scheduleFormat !== 'input'}
-                  onBlur={handleBlur}
-                  onPressEnter={handlePressEnter}
-                />
-              </div>
-            </StyledInputContainer>
-          </div>
-        </Radio.Group>
+          )}
```

Comment:
Same as above - we're rendering CronPicker only when `scheduleFormat === 'picker'` so can it be disabled?